rs/ledger_suite/tests/sm-tests/src/lib.rs

```diff
@@ -2644,15 +2644,15 @@ pub fn test_upgrade_serialization(
                     // Test upgrade to memory manager again
                     test_upgrade(ledger_wasm_nextmigrationversionmemorymanager.clone());
 
-                    // Current mainnet ICP wasm (V0) cannot deserialize from memory manager, but ICRC (V1) can
+                    // Current mainnet ICP and ICRC wasms (V1) can both deserialize from memory manager
                     match env.upgrade_canister(
                         ledger_id,
                         ledger_wasm_mainnet.clone(),
                         upgrade_args.clone(),
                     ) {
                         Ok(_) => {
                             if !downgrade_to_mainnet_should_succeed {
-                                panic!("Downgrade from memory manager directly to mainnet should fail (since mainnet is V0)!")
+                                panic!("Downgrade from memory manager directly to mainnet should fail (since mainnet is V1)!")
```

Comment:
V1 can deserialize from mem mgr. Maybe the comment should say simply "Downgrade to mainnet should fail!".

Reply:
Changed as suggested!